### DISCUSSION TITLE
if specify only 1 country

### DISCUSSION THREAD

**Discussion by iwowpao** [dr5hn/countries-states-cities-database#250] [Ideas]
can you please share sample code if I only need Brazil state and cities?  how can I do?

**Comment by dr5hn** [comment]
@iwowpao That's a really good question.

I was thinking to build a tool that allows you to select and export the state and cities of your want. But that's something gonna take a while.

However, you can request an API Key here. 
https://countrystatecity.in/request

And you can make the following requests to get the output.
https://api.countrystatecity.in/v1/countries/BR/states -- This will fetch all states under brazil
https://api.countrystatecity.in/v1/countries/BR/cities -- This will fetch all cities under brazil

**Comment by iwowpao** [comment]
I tried the index.html under /docs and it is exactly how I want to demo.
What if without API, how can I simulate as BRAZIL is clicked?
I am not sure if I know how to use your APIs to integrate into my form.

**Comment by iwowpao** [comment]
In most of cases, a site may need only few countries (state/city) options.  If the tool can be capable to do multiple select and export the states/city, that will be much great.  I am very new to JS, otherwise I would love to create more options based on your code.  It is a very good tool and thank you for sharing.

**Comment by yeahfine** [comment]
I would like to know on what basis you have precisely categorised 250 countries, 5,038 states and 151,024 cities?
